aiida/common/log.py

```diff
@@ -57,7 +57,9 @@ def emit(self, record):
             return
 
         if not hasattr(record, 'objpk'):
```

Comment:
Does this logic still make sense? The record should no longer have a `objpk` correct?

Reply:
You are right on this. The idea is to avoid storing log records of unstored nodes.
However this check is done in create_entry_from_record and for this reason we need the objpk in these methods (an unstored node doesn't have an objpk)

Reply:
Wouldn't it be better perhaps, to get rid of the `objpk` completely and adapt the `get_dblogger_extra` function to *only* return a dictionary (containing the `name` and `uuid` of the entity`) if the entity is stored. The `emit` method can then simply check for the presence of both the `objuuid` and `objname` key in the record and if they exist, we know the entity is stored.

Reply:
I did this change too even if I believe that this approach is also sub-optimal. Thanks for all the comments!